LEGO1/realtime/orientableroi.h

```diff
@@ -16,34 +16,29 @@ class OrientableROI : public ROI {
 		ZEROVEC3(m_world_velocity);
 		IDENTMAT4(m_local2world.GetMatrix());
 	}
-	// SYNTHETIC: LEGO1 0x100a4630
-	// OrientableROI::`scalar deleting destructor'
 
-	virtual const Vector3& GetWorldVelocity() const;
-	virtual const BoundingBox& GetWorldBoundingBox() const;
-	virtual const BoundingSphere& GetWorldBoundingSphere() const;
-
-protected:
-	// vtable + 0x14
-	virtual void VTable0x14() { VTable0x1c(); }
-	virtual void UpdateWorldBoundingVolumes() = 0;
+	virtual const Vector3& GetWorldVelocity() const;                // vtable+0x8
```

Comment:
Needs a bunch of `override` keywords. `//vtable` annotations in the parent class would also be good.